app/src/main/java/org/hackillinois/android/custom/CustomRefreshView.java

```diff
@@ -0,0 +1,164 @@
+package org.hackillinois.android.custom;
+
+import android.animation.ValueAnimator;
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.RectF;
+import android.support.v4.content.ContextCompat;
+import android.util.AttributeSet;
+import android.view.View;
+import android.view.animation.LinearInterpolator;
+
+import com.dinuscxj.refresh.IRefreshStatus;
+
+import org.hackillinois.android.R;
+
+/**
+ * Copied from library: changed it to have right color scheme
+ */
+public class CustomRefreshView extends View implements IRefreshStatus {
+    private static final int MAX_ARC_DEGREE = 330;
+    private static final int ANIMATION_DURATION = 888;
+    private static final int DEFAULT_START_DEGREES = 285;
+    private static final int DEFAULT_STROKE_WIDTH = 2;
+
+    private final RectF mArcBounds = new RectF();
```

Comment:
I'm not a huge fan of starting variables with "m". I know it's copied from a library but it would be nice if it looked the same as the rest of our code.

Reply:
Feel free to change it or not, but I'm going to accept this PR because everything else looks good otherwise!